iothub/device/src/Twin/ClientTwin.cs

```diff
@@ -0,0 +1,27 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.Azure.Devices.Client
+{
+    /// <summary>
+    /// A container for client properties retrieved from the service.
+    /// </summary>
+    public class ClientTwin
+    {
+        internal ClientTwin(DesiredPropertyCollection requestsFromService, ReportedPropertyCollection reportedByClient)
```

Comment:
protected internal for unit testability?

Reply:
still just internal

Reply:
updated